fastapi_pagination/api.py

```diff
@@ -361,7 +361,7 @@ def _update_route(route: APIRoute) -> None:
         ),
     )
 
-    route.body_field = get_body_field(dependant=route.dependant, name=route.unique_id)
+    route.body_field = get_body_field(flat_dependant=route.dependant, name=route.unique_id, embed_body_fields=True)
```

Comment:
The changes is not compatible with FastAPI < 0.112.4. Maybe we need to keep compatibility with previous versions of FastAPI and check the FastAPI version here?

Reply:
I have committed a change that should assure backwards compatibility for fastapi < 0.112.4